dpdata/stat.py

```
@@ -116,15 +116,15 @@ class Errors(ErrorsBase):
     SYSTEM_TYPE = LabeledSystem
 
     @property
-    @lru_cache()
+    @lru_cache
```

Ruff (0.8.2)

Use of `functools.lru_cache` or `functools.cache` on methods can lead to memory leaks

(B019)